cmd/state/internal/cmdtree/get.go

```diff
@@ -0,0 +1,46 @@
+package cmdtree
+
+import (
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runners/get"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+func newGetCommand(prime *primer.Values) *captain.Command {
+	params := &get.Params{
+		Namespace: &project.Namespaced{AllowOmitOwner: true},
+	}
+
+	cmd := captain.NewCommand(
+		"get",
+		"",
+		locale.Tl("get_description", "Checkout the given project and setup its runtime"),
```

Comment:
Should this be `get` the given project?

Reply:
No, I asked about that, but Nathan said to keep the current messaging that uses "checkout".